sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/sources/GeneratorConfig.java

```diff
@@ -186,6 +187,46 @@ public long estimatedBytesForEvents(long numEvents) {
            + numBids * configuration.avgBidByteSize;
   }
 
+  public int getAvgPersonByteSize() {
```

Comment:
<!--new_thread; commit:dd2591661935e2ce1c9472d2dd9c401e3011ec9a; resolved:1-->
This kind of thing is a very good candidate for `@AutoValue`